site/docs/tutorials/pulsar.md

```diff
@@ -384,8 +375,8 @@ If `MessagePublisher` was running while you were following these steps,
 you'll now get a report on the whole history of the events and then a
 steady stream of real-time updates.
 
-Sample output:
-Between 12:44:30:000 - 12:45:30:000 Pulsar got 522 messages from user4.
+Sample output: 12:44:30:030 Between 12:44:30:000 - 12:45:30:000 Pulsar
```

Comment:
To have it aligned with timestamps in following example output it should be `10:38:44.504  Between 10:32:00:00-10:32:30:000 Pulsar got 508 messages from user4.`

Reply:
I have aligned it.